bin/mergeq_common.sh

```diff
@@ -0,0 +1,14 @@
+project_dir=".mergeq"
+hooks_dir="$project_dir/hooks"
+
+function run_hook {
+  hook_name=$1
+  hook="$hooks_dir/$hook_name"
+
+  [[ -f $hook ]] || return 0
+
+  status "Running hook: $hook_name..."
+
+  eval "$hook \"$target_branch\" \"$merge_branch\""
+  [[ $? -eq 0 ]] || exit $?
+}
```

Comment:
I wonder if you should merge this back in and maybe surround it w/ comments so we know it is shared, just to keep the scripts self contained.